app/javascript/controllers/index.js

```diff
@@ -1,8 +1,15 @@
 import { Application } from "@hotwired/stimulus"
 import { Autocomplete } from 'stimulus-autocomplete'
+import NestedForm from 'stimulus-rails-nested-form'
+import OrderedForm from './ordered_form_controller'
 import { definitions } from 'stimulus:./'
 
 const application = Application.start()
 application.load(definitions)
 
 application.register('autocomplete', Autocomplete)
+application.register('author-form', OrderedForm)
```

Comment:
Because of double nesting, each nested form controller must be separately registered with a unique name.